### PR TITLE
Fix/show toggler empty evaluation

### PR DESCRIPTION
**Pull request by ulferts**
https://community.openproject.org/work_packages/18195

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://fix-show-toggler-empty-evaluation.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://fix-show-toggler-empty-evaluation.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: frontend/tests/unit/tests/work_packages/controllers/details-tab-overview-controller-test.js [hunk 1]
```diff
@@ -720,5 +720,47 @@ describe('DetailsTabOverviewController', function() {
...
+
+    describe('anyEmptyWorkPackageValue', function() {
+      describe('with a group having empty attributes', function() {
+        before(function() {
+          scope.groupedAttributes = [ { attributes: [ { value: 'a' }, { value: null } ] },
+                                      { attributes: [ { value: 'b' }, { value: 'c' } ] } ];
+        });
+
+        it('is true', function() {
+          expect(scope.anyEmptyWorkPackageValue()).to.eq(true);
```
Comment (myabc) [inline frontend/tests/unit/tests/work_packages/controllers/details-tab-overview-controller-test.js]: @ulferts minor, but I prefer using `to.be.true` chain.

Comment (ulferts) [inline frontend/tests/unit/tests/work_packages/controllers/details-tab-overview-controller-test.js]: @myabc simply didn't know it exists so thanks for letting me know.

**Comment by myabc** [comment]
:+1:

**Pull request merged**